roles/openshift_storage_glusterfs/tasks/glusterblock_deploy.yml

```diff
@@ -1,4 +1,17 @@
 ---
+- name: Create heketi block secret
+  oc_secret:
+    namespace: "{{ glusterfs_namespace }}"
+    state: present
+    name: "heketi-{{ glusterfs_name }}-admin-secret-block"
+    type: "gluster.org/glusterblock"
+    force: True
+    contents:
+    - path: key
+      data: "{{ glusterfs_heketi_admin_key }}"
+  when:
+  - glusterfs_heketi_admin_key is defined
+
```

Comment:
Move all the above to `glusterfs_common.yml` and place it after the include of glusterblock_deploy`. Also make it conditional on `openshift_storage_glusterfs_block_storageclass`.